soknad-felles/src/main/java/no/nav/k9/søknad/felles/Landkode.java

```diff
@@ -1,13 +1,23 @@
 package no.nav.k9.søknad.felles;
 
+import java.util.Objects;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
 
-import java.util.Objects;
-
+/**
+ * ISO 3166 alpha-3 landkode.
+ * 
+ * @see https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
```

Comment:
Hørte med Frode, ikke disse landkodene som sendes nå, men så lenge ikke denne validatoren brukes mot Pleiepengesøkaden enda så har det jo ikke nå å si - men bør sjekkes ut når Pleiepengesøknad skal inn i k9-sak

Reply:
hvilke landkoder sender dere? Vi takler både 2 og 3 bokstavskodene her, men 3 bokstavskoder  bør foretrekkes når vi kan velge (2 bokstavskoder er eldre kodeverk)

Reply:
i testcase for pleiepengerbarn benyttes 3-bokstavskode POL (Polen).